kafka_utils/util/client.py

```diff
@@ -59,3 +67,86 @@ def send_offset_commit_request_kafka(
 
         return [resp if not callback else callback(resp) for resp in resps
                 if not fail_on_error or not self._raise_on_response_error(resp)]
+
+    def send_consumer_metadata_request(self, payloads=[], fail_on_error=True,
+                                       callback=None):
```

Comment:
This is simply calling KafkaToolProtocol instead of KafkaProtocol. The real change is in KafkaToolProtocol which actually has working encode_consumer_metadata_request and decode_consumer_metadata_request methods compared to the upstream ones that don't actually work anymore. The changes we'd care about would be in kafka_utils/util/protocol.py but I can put a comment making that explicit